src/Digdir.Domain.Dialogporten.WebApi/Program.cs

```diff
@@ -200,7 +203,27 @@ static void BuildAndRun(string[] args)
             uiConfig.DocumentPath = dialogPrefix + "/swagger/{documentName}/swagger.json";
         });
     app.MapControllers();
-    app.MapHealthChecks("/healthz");
+    app.MapHealthChecks("/startup", new HealthCheckOptions
+    {
+        Predicate = check => check.Tags.Contains("dependencies"),
+        ResponseWriter = UIResponseWriter.WriteHealthCheckUIResponse
+    });
+    app.MapHealthChecks("/liveness", new HealthCheckOptions
+    {
+        Predicate = check => check.Tags.Contains("self"), // Retains the self check for liveness
+        ResponseWriter = UIResponseWriter.WriteHealthCheckUIResponse
+    });
+    app.MapHealthChecks("/readiness", new HealthCheckOptions
+    {
+        Predicate = check => check.Tags.Contains("dependencies"),
```

Comment:
Hm, readiness now includes well-known, could it be in startup only? 
I think they are only loaded once? (Cached for the lifetime of the application)